src/ROUTES.js

```diff
@@ -85,6 +85,8 @@ export default {
     MONEY_REQUEST_CONFIRMATION: ':iouType/new/confirmation/:reportID?',
     MONEY_REQUEST_CURRENCY: ':iouType/new/currency/:reportID?',
     MONEY_REQUEST_DESCRIPTION: ':iouType/new/description/:reportID?',
+    MONEY_REQUEST_MANUAL_TAB: ':iouType/new/:reportID?/manual',
```

Comment:
I'm a bit skeptical of this ... are you allowed to have optional URL segments followed by mandatory ones?

Reply:
I was too, but It seems to work as intended!